recipe/meta.yaml

```
@@ -27,25 +27,40 @@ requirements:
     - yarl
     - urllib3 <2  # [py<310]
```

```suggestion
    - urllib3 <2  # [py<310]
    - urllib3        # [py>=310]
```
see https://github.com/kevin1024/vcrpy/blob/v7.0.0/setup.py#L44